src/lib/components/ExpertDataCard.tsx

```diff
@@ -14,11 +14,12 @@ export interface IExpertDataCardProps {
 const ExpertDataCard: React.FC<IExpertDataCardProps> = (props) => {
   const { expert } = props;
 
-  const fullName = `${expert.firstName} ${expert.secondName}`;
+  const fullName = expert && `${expert.firstName} ${expert.secondName}`;
```

Comment:
remove `expert && ` conditions everywhere since `expert` is required now

Reply:
fixed